src/components/Presentation/index.jsx

```diff
@@ -31,8 +33,26 @@ class Presentation extends React.Component {
   addSlide = () => {
     this.props.dispatch({type: 'CREATE_SLIDE', payload: {title: 'Title', subtitle: 'Subtitle'}});
   }
+  updateCanvas() {
+    let canvas = new fabric.Canvas(this.canvas.current);
+    let currentSlide = this.props.slides.toJS().filter((slide, index) => {if (index === this.state.currentSlide) return slide});
+    let title = new fabric.Text(currentSlide[0].title, {
```

Comment:
Same thing here, if you try to view one slide and then go back we don't persist the location of the text nor the canvas. This isn't a problem at the moment, but we should really serialize the canvas and store it for the particular slide, and reload it to the canvas when the user clicks on that specific slide.